src/queueing_honey_badger.rs

```diff
@@ -172,6 +171,20 @@ where
     queue: TransactionQueue<T>,
 }
 
+impl<T, N> Debug for QueueingHoneyBadger<T, N>
+where
+    T: Contribution + Serialize + for<'r> Deserialize<'r> + Debug,
+    N: NodeIdT + Serialize + for<'r> Deserialize<'r> + Rand + Debug,
+{
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("QueueingHoneyBadger")
+            .field("batch_size", &self.batch_size)
+            .field("dyn_hb", &self.dyn_hb)
+            .field("queue", &self.queue)
+            .finish()
```

Comment:
Isn't that precisely the derived implementation?

Reply:
Sorry, I forgot to derive it after removing an `rng` filed from `QueueingHoneyBadger`.